contracts/12-TimeLock/TokenTimeLock.sol

```diff
@@ -45,79 +50,101 @@ contract TokenTimeLock {
     if (addressCheck == false) {
       addresses.push(msg.sender);
     }
-    // Omrekenen tijd in weken naar seconden
+    //Convert the time from weeks to seconds
     uint increase = _increase.mul(604800);
     // updated locktime 1 week from now
+
+    //Check if the address already has a locktime, if not add the locktime to the respected address.
+
     if (lockTime[msg.sender] == 0) {
       lockTime[msg.sender] = block.timestamp.add(increase);
     } else {
       lockTime[msg.sender] = lockTime[msg.sender].add(increase);
     }
-    // Transfer THX naar contract voor staken
+    // Transfer THX to contract for staking
     THXtoken.transferFrom(msg.sender, address(this), amount);
-    // Transfer stTHX naar User
+    // Transfer stTHX to User
     stTHXtoken.approve(address(this), amount);
     stTHXtoken.transferFrom(address(this), msg.sender, amount);
 
-    // Log hoeveel gestaked is
+    // Log how much was deposited 
     emit Staked(msg.sender, amount);
   }
 
-  function getAddress() public view returns (address[] memory) {
+  //see which addreses are stored in the addresses array
+
+  function getAddress() public view returns (address[] memory) onlyAdmin{
     return addresses;
   }
 
+  //see which address has a locktime
+
   function getLocktime() public view returns (uint256) {
     return lockTime[msg.sender];
   }
 
   // Allocating per user the coin and the amount it has
-  function allocate(
-    address _userAddress,
-    address _tokenAddress,
-    uint256 allocating
-  ) public onlyAdmin {
-    allocations[_tokenAddress][_userAddress] = allocations[_tokenAddress][_userAddress].add(allocating);
-    emit Allocated(_tokenAddress, allocating);
-  }
-
-  // delete test for remix
+  //This function can be used in the future, it is still in development
```

Comment:
@S4WY Hey! Logic that is not currently used should be removed from this version. If we require it in the future, we will use version control to find it:) Currently dead code (not so much in this repo though) will only decrease the readability.